### DISCUSSION TITLE
Lesson 6 -failed to connect to the new network on metamask

### DISCUSSION THREAD

**Discussion by MahammadKumar** [Cyfrin/foundry-full-course-cu#140] [Q&A] [answered]
I have encountered an error while attempting to establish a connection to the local environment using the RPC URL: https://127.0.0.1:8545/ and the chain ID: 31337 in metamask. Specifically, the error message states, "Could not fetch chain ID. Is your RPC URL correct?"

I kindly request assistance from anyone who can provide guidance or support in resolving this issue.

**Comment by alymurtazamemon** [comment, marked as answer]
@MahammadKumar Please check this out #37
